.github/CODEOWNERS

```diff
@@ -1,5 +1,24 @@
 # This a CODEOWNERS file, where you can establish code owners.
 # Code owners are automatically requested for review when someone opens a pull request
-# that modifies code that they own. In this instance, cicdguy will be notified of any
-# changes made the files inside the .github/workflows directory.
-.github/workflows/* @cicdguy
+# that modifies code that they own.
+
+# admiral.test
+data-raw/ds.R @Gopi2345
+data-raw/lb.R @yangc71
```

Comment:
do we need to give some of these people access to the repo? otherwise the codeowners file will have errors.

Reply:
it seems so, could you please grant them access if they are okay to be listed as code owners? thanks!